packs/data/kingmaker-bestiary.db/svetlana.json

```diff
@@ -108,6 +108,11 @@
         },
         {
             "_id": "TphEXfmeJVfSPoXQ",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.WmSZELnHCZ9g9Nq2.Item.TphEXfmeJVfSPoXQ"
```

Comment:
We don't want in world references in the compendium packs. Make sure you link to compendium copies only.

Reply:
Was this item dragged from another actor?

Reply:
Nope, was just selected from the Actor list and then exported back to the Compendium afterwards. I can mess around with it again later tonight and get it cleaned up, just confused what caused it so that I don't do it again. At least I know now to keep an eye out for it going forward.